docs/resources/blocks-and-transactions.md

```diff
@@ -0,0 +1,1560 @@
+| Return From "gov" Module Account | `Base64Decode(.result.end_block_events[event_index].attributes[attribute_index].value) === "{gov module account}"` <br />where <br />`.result.end_block_events[event_index].type === "transfer" && Base64Decode(.result.end_block_events[event_index].attributes[attribute_index].key) === "sender"`. | String                        |
+| Return To Address                | `Base64Decode(.result.end_block_events[event_index].attributes[attribute_index].value)` <br />where <br />`.result.end_block_events[event_index].type === "transfer" && Base64Decode(.result.end_block_events[event_index].attributes[attribute_index].key) === "recipient"`.                         | String                        |
+| Returned Deposit Amount          | `Base64Decode(.result.end_block_events[event_index].attributes[attribute_index].value)` <br />where <br />`.result.end_block_events[event_index].type === "transfer" && Base64Decode(.result.end_block_events[event_index].attributes[attribute_index].key === "amount"`).                            | [Asset String](#asset-string) |
+
+Note that there may be multiple depositors of a proposal, so the event may appears multiple times.
+
+```json
+[
+  {
+    "type": "transfer",
+    "attributes": [
+      {
+        "key": "recipient",
+        "value": "cro15d0esada05wx4lpncp5vmwmzgrcnu3cu5xfy9w",
+        "index": true
+      },
+      {
+        "key": "sender",
+        "value": "cro10d07y265gmmuvt4z0w9aw880jnsr700jzemu2z",
+        "index": true
+      },
+      {
+        "key": "amount",
+        "value": "1000000000basecro",
+        "index": true
+      }
+    ]
+  },
+  {
+    "type": "transfer",
+    "attributes": [
+      {
+        "key": "recipient",
+        "value": "cro1meq5gd2fwjcav8n0nynwelusm9hrngwglvp426",
+        "index": true
+      },
+      {
+        "key": "sender",
+        "value": "cro10d07y265gmmuvt4z0w9aw880jnsr700jzemu2z",
+        "index": true
+      },
+      {
+        "key": "amount",
+        "value": "2000000000000basecro",
+        "index": true
+      }
+    ]
+  }
+]
+```
+
+[Top](#table-of-content)
+
+### 2c. Burn Proposal's Deposit
+
+There are a few cases where a proposal deposits will be burnt. For details, plesae refer to [Proposal Result](#4-proposal-result) for more details.
+
+If a proposal does not get enought votes that exceed the "quorum" or has the "No with Veto" votes exceed the "veto_threshold", the deposit will **NOT** be returned to the depositors. Those deposits **will be burnt** from the "gov" module account as well.
+
+To monitor a proposal got rejected and deposits got burned, it can be detected by monitoring the `end_block_events` in Tendermint Block Results API. There will be an "proposal_result" event marking the proposal as rejected, and different from [Return Proposal's Deposit](#2b-return-proposals-depsoit), there will be **NO** transfer event in the `end_block_results`, which means the deposits is **NOT** returned and is burnt.
```

Comment:
```suggestion
To monitor a proposal that got rejected and deposits got burned, it can be detected by monitoring the `end_block_events` in Tendermint Block Results API. There will be a "proposal_result" event marking the proposal as rejected, and different from [Return Proposal's Deposit](#2b-return-proposals-depsoit), there will be **NO** transfer event in the `end_block_results`, which means the deposits is **NOT** returned and is burnt.
```